charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -87,6 +87,12 @@ server:
   {{- end }}
 
 plugins:
+
+  {{- if .Values.credentialComposer.uniqueID.enabled }}
+  CredentialComposer:
```

Comment:
Is this extra white space intentional?

Reply:
Yeah. CredentialComposer goes under the plugins section at like 89.